meerk40t/gui/scene/scenewidgets.py

```diff
@@ -583,19 +655,72 @@ class RectSelectWidget(Widget):
     Rectangle Selection Widget, draws the selection rectangle if left-clicked and dragged
     """
 
+    # selection_method = 1 = hit, 2 = cross, 3 = enclose
+    # Color for selection rectangle (hit, cross, enclose)
+    selection_colors = [
+        wx.RED,
+        wx.GREEN,
+        wx.BLUE,
+    ]
+    # Visual help for selection rectangle (to help those with visual challenges i.e colorblindness)
+    selection_line = [wx.PENSTYLE_DOT_DASH, wx.PENSTYLE_DOT, wx.PENSTYLE_SHORT_DASH]
```

Comment:
```python
SELECTION_TOUCH = 0
SELECTION_CROSS = 1
SELECTION_ENCLOSE = 2
```
It's mostly style to make sure the code is more readable for future people since code tends to get read more times than it's written.